normandy/selfrepair/static/js/self_repair_runner.js

```diff
@@ -1,3 +1,8 @@
+import 'babel-polyfill'
+import xhr from './xhr.js'
+import Mozilla from './uitour.js'
+import Normandy from './normandy_driver.js'
```

Comment:
This file needs uuid imported too. I only get an error for this when I run this in docker-compose, not when I run it normally, which is weird.